base/setup/usetup/lang/he-IL.h

```diff
@@ -1874,37 +1873,41 @@ MUI_STRING heILStrings[] =
     "on %I64u %s  Harddisk %lu  (Port=%hu, Bus=%hu, Id=%hu) [%s]."},
     {STRING_HDINFOPARTZEROED_1,
     "Harddisk %lu (%I64u %s), Port=%hu, Bus=%hu, Id=%hu (%wZ) [%s]."},
+    // {STRING_HDINFOPARTZEROED_2,
+    // "Harddisk %lu (%I64u %s), Port=%hu, Bus=%hu, Id=%hu [%s]."},
     {STRING_HDDINFOUNK4,
     "%c%c  Type 0x%02X    %I64u %s"},
     {STRING_HDINFOPARTEXISTS_1,
     "on Harddisk %lu (%I64u %s), Port=%hu, Bus=%hu, Id=%hu (%wZ) [%s]."},
+    // {STRING_HDINFOPARTEXISTS_2,
+    // "on Harddisk %lu (%I64u %s), Port=%hu, Bus=%hu, Id=%hu [%s]."},
     {STRING_HDDINFOUNK5,
     "%c%c %c %sType %-3u%s                      %6lu %s"},
     {STRING_HDINFOPARTSELECT_1,
     "%6lu %s  Harddisk %lu  (Port=%hu, Bus=%hu, Id=%hu) on %wZ [%s]"},
     {STRING_HDINFOPARTSELECT_2,
     "%6lu %s  Harddisk %lu  (Port=%hu, Bus=%hu, Id=%hu) [%s]"},
     {STRING_NEWPARTITION,
-    "Setup created a new partition on"},
+    "�� ���� ����� ���� ������ �����"},
     {STRING_UNPSPACE,
     "    %sUnpartitioned space%s            %6lu %s"},
     {STRING_MAXSIZE,
-    "MB (max. %lu MB)"},
+    ")�""� %ul .���( �""�"},
     {STRING_EXTENDED_PARTITION,
-    "Extended Partition"},
+    "������ �����"},
     {STRING_UNFORMATTED,
-    "New (Unformatted)"},
+    ")������ ��( ���"},
     {STRING_FORMATUNUSED,
-    "Unused"},
+    "������ ��"},
     {STRING_FORMATUNKNOWN,
-    "Unknown"},
+    "���� ��"},
     {STRING_KB,
-    "��������"},
+    "�""�"},
     {STRING_MB,
-    "�������"},
+    "�""�"},
     {STRING_GB,
-    "��������"},
+    "�""�"},
```

Comment:
Is that really two successive quotation marks?

Reply:
_shroug_

Reply:
`\"` is not correct escape sequence here.

@HBelusca was asking only in regard of this one.

Reply:
Oh wait, it's not RC file, nevermind then. :sweat_smile:

Reply:
> @HBelusca was asking only in regard of this one.

Not only of this one but of (many?) others I see elsewhere.
Also @binarymaster don't forget these .h files are not *resource files* processed by RC the resource compiler, but authentic C source file (headers), and as such the C rules for strings should be applied.

Reply:
Acronyms in Hebrew use a quotation mark before the last letter.
GB == ג"ב
Other way is to write the entire word or to use the original "GB"
Is there something problematic in trying to put a quotation mark in that string?

Reply:
> Is there something problematic in trying to put a quotation mark in that string?

Sure not, but just use the correct escaping (since it's C header file, the standard C string rules apply, so you indeed need `\"` for escaping quotation marks inside the string).
And/or use explicit size units.